sales.html

```diff
@@ -1,26 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-
-    <title>All Stores Sales Data</title>
+    <meta charset="utf-8">
+    <title></title>
   </head>
   <body>
-    <h1>All Store Data</h1>
-    <h2>Pike</h2 class="heading">
-    <section id="pike">
-    </section>
-    <section id="seatac">
-      <h2>Seatac</h2 class = "heading">
-    </section>
-    <section id = "seacenter">
-      <h2>Seattle Center</h2  class = "heading"1>
-    </section>
-    <section id = "caphill">
-      <h2>Capitol Hill</h2 class = "heading">
-    </section>
-    <section id = "alkai">
-      <h2>Alkai</h2  class = "heading">
+    <section id="cookie-table">
+
     </section>
-    <script src="salesapp.js" type="text/javascript"></script>
+    <script src="salesapp.js" ></script>
```

Comment:
Keep the type="text/javascript" attribute. Your browser is smart enough to figure it out, but it's better to be more specific.